src/components/LeaderBoard/LeaderBoard.jsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react'
```

Comment:
@KodingKing Please keep the extention file name as .js not .jsx as we are keeping the same for all the files. Please make the same changes for all the files.